### ISSUE TITLE
Issue title no error for blank entry

### ISSUE THREAD

**Issue by hellonoko** [gitscout/gitscout-feedback#72]
Leaving a blank field for an issue name will prevent 'Submit new issue' button from working but wont highlight the field or show any type of error or reason why.

**Comment by jeremybenaim** [comment]
good catch @hellonoko. We're going to fix this in the next version! Thanks 😊
